pages/shared/studentGroupAssessmentHelpers/studentGroupAssessmentHelpers.js

```diff
@@ -0,0 +1,137 @@
+var ERR = require('async-stacktrace');
+
+var sqldb = require('../../../prairielib/lib/sql-db');
+var sqlLoader = require('../../../prairielib/lib/sql-loader');
+var sql = sqlLoader.loadSqlEquiv(__filename);
+
+module.exports.getConfigInfo = function (req, res, next) {
```

Comment:
We generally only name callback parameters `next` if a function is used as [Express middleware](https://expressjs.com/en/guide/using-middleware.html). Since this is just a function, you should name it `callback` instead.

The `req` param appears to be unused here; you should remove it.

I'd recommend naming this `getGroupInfo` or something like that instead.

Reply:
Callback fixes done in [8652798](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/86527983f782382ca918fb20f3c9b995e3862239), rename done in [12caec9](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/12caec91b31edce562c0072f8750b7466d518909)